File: 2ndPeriod/aedsII/clike/operators/operators.java
package aedsII.clike.operators;
import java.util.Scanner;

class operators {
    public static void main(String[] args){
        int x=10, y=3;
        int sum, sub, mult, remainder, quoc;
        float div; 
        Scanner scanner = new Scanner(System.in);

        sum = x+y;
        System.out.println("x + y = " + sum);
        sub = x-y;
        System.out.println("x - y = " + sub);
        div = (float)x/y;
        System.out.println("x / y = " + div);
        mult = x*y;
        System.out.println("x * y = " + mult);
        remainder = x%y;
        System.out.println("x % y = " + remainder);
        quoc = x/y;
        System.out.println("x / y = " + quoc);
        int inc = x+1;
        System.out.println(x + "++ = " + inc);
        int dec = y-1;
        System.out.println(y + "-- = " + dec);
    }
}

File: 2ndPeriod/aedsII/clike/operators/operators.cs
using System;

class MainClass{
    public static void Main(string[] args){
        int x=6, y=5;
        int sum, sub, mult, remainder, quoc, inc, dec;
        float div;

        sum = x+y;
        Console.WriteLine(x + " + " + y + " = " + sum);
        sub = x-y;
        Console.WriteLine(x + " - " + y + " = " + sub);
        mult = x*y;
        Console.WriteLine(x + " * " + y + " = " + mult);
        div = (float)x/y;
        Console.WriteLine(x + " / " + y + " = " + div);
        quoc = x/y;
        Console.WriteLine(x + " / " + y + " = " + quoc);
        remainder = x%y;
        Console.WriteLine(x + " % " + y + " = " + remainder);
        inc = ++x;
        Console.WriteLine(x-1 + "++ = " + inc);
        dec = --y;
        Console.WriteLine(y+1 + "-- = " + dec);
    }
}